actors/miner/src/lib.rs

```diff
@@ -1950,6 +1958,306 @@ impl Actor {
         Ok(ProveCommitSectors3Return { activation_results: result })
     }
 
+    fn prove_commit_sectors_ni(
+        rt: &impl Runtime,
+        params: ProveCommitSectorsNIParams,
+    ) -> Result<(), ActorError> {
+        let policy = rt.policy();
+        let miner_id = rt.message().receiver().id().unwrap();
+
+        if params.sectors.is_empty() {
+            return Err(actor_error!(illegal_argument, "batch empty"));
```

Comment:
```suggestion
            return Err(actor_error!(illegal_argument, "no sectors"));
```

Reply:
just trying to avoid "batch" here

Reply:
in fact, I think we should just remove this block entirely since it'll be caught in the policy check down further